integration_tests/test_get_requests.py

```diff
@@ -43,6 +42,19 @@ def check_response(r: Response):
     check_response(get(f"/{function_type}/template"))
 
 
+@pytest.mark.benchmark
+@pytest.mark.parametrize("function_type", ["sync", "async"])
+def test_url_for(function_type: str, session):
+    def check_response(r: Response):
+        assert r.status_code == 200
+        assert r.text == f"<a href='/{function_type}/url_for'>/{function_type}/url_for</a>"
+        assert "<!DOCTYPE html>" in r.text
+        assert "Jinja2" in r.text
+        assert "Robyn" in r.text
```

Comment:
we don't need to test the basic functionalities in this test. we have a dedicated test for that.